content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,210 @@
+:show-content:
+
+========
+My Lunch
+========
+
+When the *Lunch* application is opened, the :guilabel:`Order Your Lunch` dashboard loads. This view
+is also accessed by navigating to :menuselection:`Lunch app --> My Lunch: New Order`.
+
+Order Your Lunch dashboard
+==========================
+
+On the main :guilabel:`Order Your Lunch` dashboard, all the necessary information needed to place an
+order is visible. The default filter for the products is :guilabel:`Available Today`. This filter
+shows only products that can be purchased that day.
+
+The left-side of the dashboard displays the various :guilabel:`Categories` of products available,
+along with the :guilabel:`Vendors` supplying the products. On the far-right side of each line is a
+number, which indicates how many products are associated with that respective category or vendor.
+
+To filter the products by categories or vendors, click the checkbox next to the specific category or
+vendor to view only those items.
+
+.. note::
+   If multiple selections are made, **only** products that fall under **all** the selected options
+   are shown.
+
+The top portion of the dashboard, the :guilabel:`Your Order` summary, displays the user's account
+information, and the order details for the current day, if any orders have been placed.
+
+The main section, beneath the user's information, houses all the products in a Kanban view. Each
+product card displays the name, cost, vendor, photo, and a description of the product. If the
+product is configured as new, it also displays a :guilabel:`New` tag.
+
+.. image:: orders/dashboard.png
+   :align: center
+   :alt: The *Lunch* app dashboard with all areas highlighted.
+
+.. note::
+   Anywhere the vendor is listed in the *Lunch* app, such as the Kanban product cards, both the name
+   of the vendor appears as well as their phone number.
+
+The products can be displayed in a list view by clicking the :guilabel:`≣ (four parallel lines)`
+icon in the top-right.
+
+Placing orders
+==============
+
+To place a lunch order, navigate to the main :guilabel:`Order Your Lunch` dashboard.
```

Comment:
I just jumped to this section from the top-- how do I get to the main dashboard--- menu selection..path